content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -97,9 +97,10 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    ```
    Or use this for detailed view of version:
```

Comment:
Preview: https://deploy-preview-32515--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#:~:text=Some%20More%20About%20Output%20Options

IMO, the indentation of newly added text is not appropriate.
![image](https://user-images.githubusercontent.com/46767780/160258919-434affad-08d9-4673-bac8-1a33237a6b0d.png)

How about this?
```suggestion
   Or use this for detailed view of version: ([More about output options](/docs/reference/kubectl/#output-options))
```